src/components/HOCs/WithWidgetWorkspaces/WithWidgetWorkspaces.js

```diff
@@ -311,30 +312,22 @@ export const WithWidgetWorkspacesInternal = function(WrappedComponent,
      * there is no active configuration
      */
     currentConfiguration = configurations => {
-      let currentWorkspace = configurations[this.state.currentConfigurationId]
-
-      // if no current workspace, find last selected workspace from a previous session
-      if (!currentWorkspace) {
-        currentWorkspace = _find(configurations, configuration => {
-          return configuration.active && !configuration.isBroken;
-        })
-      }
-
-      // If no previously active workspace, or it's broken, find a working one
+      let currentWorkspace =
+        configurations[this.state.currentConfigurationId] ||
+        _find(configurations, ({ active, isBroken }) => active && !isBroken) ||
+        _find(configurations, ({ isBroken }) => !isBroken) ||
+        this.state.defaultWorkspace
```

Comment:
Is there a reason to use lodash's `_find` here instead of `Array.prototype.find`? I've noticed a lot of use of lodash in the codebase and have been wondering about whether that's still the preferred style (now that JS has a lot of these functions natively).

Reply:
Cool, thanks. To be clear, no need to make any changes to this PR, I was just curious.